src/Blazored.Modal/BlazoredModalInstance.razor

```diff
@@ -41,7 +41,7 @@ else
 
         <div class="blazored-modal-overlay" @onclick="HandleBackgroundClick"></div>
 
-        <div id="_@Id.ToString("N")" class="@Class @AnimationClass">
+        <div id="_@Id.ToString("N")" class="@Class @AnimationClass @ScrollableClass">
```

Comment:
My question here is what is stopping developers providing an equivalent of the `blazored-modal-scrollable` class you've proposed in this PR, and just passing it to the current `Class` parameter? This would produce the same behaviour as what you're proposing and would require no changes to the library as far as I can see.

Reply:
Using the custom css is a way to work around the issue without making a change, however the developer has to work out what that specific CSS is and then add it to their project.

As a developer it would be easier for if I saw it as an option, then realise I may have content that needs to scroll and just set the option flag and all fixed.

Reply:
Fair point, a fair amount of new developers are coming from desktop or non-web backgrounds and they might not want to learn CSS or at least might be very new to it.

I would prefer that we don't add yet another variable for yet another class as I'm getting the impression this might not be the end of adding in CSS for developers. We already have the animation variable as well. This logic can be incorporated into the `SetClass` method and then we can produce a single string of CSS classes to apply.

Reply:
I was thinking the same as well, I could see in the future there might be more being added in that area.
I will update the pull request to use the SetClass method.